src/Admin/Test/Test.js

```diff
@@ -285,6 +278,7 @@ class Test extends React.Component {
                         })
                     }
                 } else {
+                    // debugger
                     if (extra.props.getTechnology && extra.props.getTechnology.length !== 0) {
```

Comment:
you can check directly `extra.props.getTechnology.length` no need to check `!==0`